docs/run-a-node/en/how-to-get.md

```diff
@@ -0,0 +1,55 @@
+---
+id: how_to_get
+title: Get Conflux Client
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/run-a-node/en/how-to-get.md
+keywords:
+  - node
+---
+
+[Conflux-rust](https://github.com/conflux-chain/conflux-rust) is a Conflux protocol high-performance implementation developed by the foundation with the Rust language. It is an Conflux Client program that runs Conflux network nodes and provides RPC service to the public. It is also a command-line tool that manages local accounts and conducts PRC interaction with nodes. You can download the release package through any of the ways listed: 
+
+## Directly download the nodes program
+
+Conflux-rust on Github [Release](https://github.com/Conflux-Chain/conflux-rust/releases) page provides a compiled node program that can be directly downloaded and run. There are two versions for the main-net and the test-net: the main-net version is Conflux `vx.x.x`, while the version for test-net is `Conflux vx.x.x-testnet` (which has a suffix of `testnet`). Each program will be provided for Linux, Windows10, and macOS.
```

Comment:
> Conflux-rust on Github [Release](https://github.com/Conflux-Chain/conflux-rust/releases) page provides a compiled node program that can be directly downloaded and run.

The [release](https://github.com/Conflux-Chain/conflux-rust/releases) page on the conflux-rust GitHub repository providers pre-built binaries that you can download and run directly.

> main-net, test-net

mainnet, testnet